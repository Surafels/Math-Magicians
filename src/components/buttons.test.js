import React from 'react';
import renderer from 'react-test-renderer';
import InputButton from './button';

test('Should render buttons correctly', () => {
  const onClick = jest.fn();

  const tree = renderer
    .create(<InputButton value="Test" onClick={onClick} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
