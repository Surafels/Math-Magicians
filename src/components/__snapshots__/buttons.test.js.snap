// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render buttons correctly 1`] = `
<input
  className={null}
  onClick={[Function]}
  type="button"
  value="Test"
/>
`;
